geanylua/glspi_app.c

```diff
@@ -416,7 +416,7 @@ static gint glspi_keycmd(lua_State* L)
 
 static gint glspi_launch(lua_State* L)
 {
-	gint argc=lua_gettop(L);
+	gsize argc=lua_gettop(L);
```

Comment:
not sure I get that one, but I believe there is a fairly serious bug in the alloc itself, shouldn't it be
`argv=g_malloc0(sizeof(gchar *)*(argc+1))` (notice the parentheses)?
Maybe we get away with it because of alignment or something, but IIUC it would likely allocate 7 bytes too few.

Reply:
Oh, right, I didn't pay enough attention. `gsize` seemed to "fix" the problem but the missing parentheses are the real problem here. Fixed now.